package.json

```diff
@@ -165,12 +174,10 @@
   "lint-staged": {
     "*.js": [
       "xo --fix",
-      "prettier --write",
-      "git add"
+      "prettier --write"
```

Comment:
Won't removing the `git add` prevent the `prettier` changes from being committed?

Reply:
Nope! The latest version of `lint-staged` does that automatically be default. It actually told me to remove this line.

Reply:
Ok